### PR TITLE
Adds IsoDateFilter

### PR DESCRIPTION
**Pull request by luxcem**
Adds an IsoDateFilter, following the IsoDateTimeFilter

### CODE REVIEW

File: django_filters/filters.py [hunk 1]
```diff
@@ -332,6 +334,13 @@ class IsoDateTimeFilter(DateTimeFilter):
     field_class = IsoDateTimeField
 
 
+class IsoDateFilter(DateTimeFilter):
```
Comment (rpkilby) [inline django_filters/filters.py]: Should this inherit from `DateFilter`?

Comment (luxcem) [reply django_filters/filters.py]: Oh yes must be a Typo

**Comment by luxcem** [comment]
So what do you think?

**Comment by carltongibson** [comment]
I'm going to say no to this. It's a nice example and easy enough to include in your own project but I have to draw the line on included filters somewhere. 

Thanks for the input.

**Pull request closed without merging**